WordPress/WordPressIntents/en.lproj/Sites.strings

```diff
@@ -1,12 +1,17 @@
+/* This text is used when the user is configuring the iOS widget and selecting which WordPress site they want the widget to be for */
 "BOl9KQ" = "There are ${count} options matching ‘${site}’.";
 
+/* This text is used when the user is configuring the iOS widget, as a label for the dropdown to select the site to configure it for */
 "ILcGmf" = "Site";
 
+/* This text is used when the user is configuring the iOS widget, as the title for the modal when selecting the site to configure it for */
 "cyajMn" = "Site";
 
+/* This text is used when the user is configuring the iOS widget */
 "gpCwrM" = "Select Site";
```

Comment:
Not sure when this one is used tbh, hence why I kept this vague.
I thought at first that it could be the text shown when you haven't selected any site yet, so I tried to add the Widget in my phone… but it got immediately pre-configured with the +platform9p2 site (not sure if that's because that's the default site I have set in WordPress, or because I previously configured the Widget when I tried the feature a while ago), so not sure that string would be for that case after all…?